examples/source_file_gpx_3d.html

```diff
@@ -72,18 +72,18 @@
                         out: {
                             crs: view.referenceCrs,
                             structure: '3d',
-                            style: {
-                                stroke : {
+                            style: new itowns.Style({
+                                stroke: {
                                     color: 'red',
                                     width: 2,
                                 },
-                                point : {
+                                point: {
                                     color: 'white',
                                 }
-                            }
+                            }),
                         }
                     }))
-                    .then(itowns.Feature2Mesh.convert({style}))
+                    .then(itowns.Feature2Mesh.convert())
```

Comment:
this modification couldn't have been done directly in https://github.com/iTowns/itowns/pull/2006 ?
(because you need this https://github.com/iTowns/itowns/pull/2135 maybe ?)

Reply:
It's a bug that seems to have appeared in a sous PR of the #2006 and as the funcitonal test is currently not well done, it got invisible.